Makefile

```diff
@@ -548,7 +548,7 @@ check-docker-hashes-consistency: $(DOCKERFILE_FROM_CHECKER)
 yetus:
 	@echo Running yetus
 	mkdir -p yetus-output
-	docker run --rm -v $(CURDIR):/src:delegated,z ghcr.io/apache/yetus:0.15.1 \
+	docker run --rm -v $(CURDIR):/src:delegated,z docker.io/lfedge/eve-yetus:latest \
```

Comment:
@rene to be honest using 'latest' is not  a good idea. we will not be able to track changes besides if we ever intruduce incomaptibility e.g. in linter settings "old" branches may stat failing eventually